tests/ui/lint/deny-inside-forbid-ignored.rs

```diff
@@ -0,0 +1,16 @@
+/// Ensure that using deny inside forbid is treated as a no-op,
+/// and does not override the level to deny.
```

Comment:
Suggestion: I would also like to see this expanded to cover cli lint level + interactions with `warnings`, i.e.

```rs
//! Ensure that using deny inside forbid is treated as a no-op, and does not override the level to
//! deny.

//@ revisions: in_source cli_forbid cli_forbid_warnings
//@[cli] compile-flags: -F unsafe_code
//@[cli] compile-flags: -F warning
```

Reply:
feel free to push it when you've implemented it already :3

Reply:
I pushed the tests, though I didn't look at the lint docs failure.